src/main/java/com/kodilla/ecommercee/ProductController.java

```diff
@@ -1,26 +1,37 @@
 package com.kodilla.ecommercee;
 
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RestController;
+
 import java.util.ArrayList;
 import java.util.List;
 
+@RestController
+@RequestMapping("v1/product")
```

Comment:
tutaj zmieniłabym na v1/project/products

Reply:
Wrócę w dom i zmienię. W sumie dobrze by było to ujednolicić